server/adaptors/opensearch_observability_plugin.ts

```diff
@@ -18,124 +17,101 @@ export function OpenSearchObservabilityPlugin(Client: any, config: any, componen
   Client.prototype.observability = components.clientAction.namespaceFactory();
   const observability = Client.prototype.observability.prototype;
 
-  observability.getPanels = clientAction({
+  // Get Object 
+  observability.getObject = clientAction({
     url: {
-      fmt: OPENSEARCH_PANELS_API.GET_PANELS,
+      fmt: OPENSEARCH_PANELS_API.OBJECT,
       params: {
-        fromIndex: {
-          type: 'number',
+        objectId: {
+          type: 'string',
+        },
+        objectIdList: {
+          type: 'string',
+        },
+        objectType: {
+          type: 'string',
+        },
+        sortField: {
+          type: 'string',
+        },
+        sortOrder: {
+          type: 'string',
+        },
+        fromIndex:{
+          type: 'string',
         },
         maxItems: {
           type: 'number',
         },
-      },
-    },
-    method: 'GET',
-  });
-
-  observability.createPanel = clientAction({
-    url: {
-      fmt: OPENSEARCH_PANELS_API.PANEL,
-    },
-    method: 'POST',
-    needBody: true,
-  });
-
-  observability.getPanelById = clientAction({
-    url: {
-      fmt: `${OPENSEARCH_PANELS_API.PANEL}/<%=panelId%>`,
-      req: {
-        panelId: {
+        name: {
           type: 'string',
-          required: true,
         },
-      },
-    },
-    method: 'GET',
-  });
-
-  observability.updatePanelById = clientAction({
-    url: {
-      fmt: `${OPENSEARCH_PANELS_API.PANEL}/<%=panelId%>`,
-      req: {
-        panelId: {
+        lastUpdatedTimeMs: {
```

Comment:
forgot to mention `createdTimeMs` is also supported, format is the same as `lastUpdatedTimeMs`